app/controllers/rating_questions_controller.rb

```diff
@@ -1,5 +1,70 @@
+# require 'pry'
+
 class RatingQuestionsController < ApplicationController
+  before_action :set_rating_question, only: [:show, :update, :destroy]
+
   def index
     @rating_questions = RatingQuestion.all
   end
+
+  def new
+    @rating_question = RatingQuestion.new
+  end
+  
+  def question_params
+    params.require(:rating_question).permit(:title, :tag)
+  end
+
+  def create
+   if request.body.size.zero?
```

Comment:
This is a good check to do. I think the `question_params` method will catch this however, and in production it will return the same status code. In `development` & testing you'll probably see an `ActionController::ParameterMissing` exception raised.

Reply:
hmm, can we speak about this during 1-1?